dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/PolicyExecutorImpl.java

```diff
@@ -1542,6 +1542,10 @@ public void updateConfig(Map<String, Object> props) {
         long u_startTimeout = null == (v = props.get("startTimeout")) ? -1l : (Long) v;
         boolean useVirtualThreads = null == (v = props.get("virtual")) ? false : (Boolean) v;
 
+        if (!virtualThreadOps.isVirtualThreadCreationEnabled()) {
+            useVirtualThreads = false;
+        }
+
         // Validation that cannot be performed by metatype:
         if (useVirtualThreads && !virtualThreadOps.isSupported()) {
             throw new IllegalArgumentException("virtual: true");
```

Comment:
I think this is what we want: Continue to always raise the exception for setting virtual=true on a JDK level that is incapable of virtual threads (The override isn't supposed to be a way to allow bad config on Java 17).  When the JDK supports virtual threads, then allow it to be overridden by the override.

```suggestion
        // Validation that cannot be performed by metatype:
        if (useVirtualThreadOps) {
            if (!virtualThreadOps.isSupported()) {
                throw new IllegalArgumentException("virtual: true");
            } else (!virtualThreadOps.isVirtualThreadCreationEnabled()) {
                useVirtualThreads = false;
            }
```

Reply:
Fixed...   running new build